apps/storybook/src/Utilities.mdx

```diff
@@ -405,6 +405,16 @@ writing your own geometries:
 - `ScatterPointsGeometry`, used in [`ScatterPoints`](https://h5web-docs.panosc.eu/?path=/docs/visualizations-scattervis--docs) (undocumented)
 - `SurfaceMeshGeometry`, used in [`SurfaceMesh`](https://h5web-docs.panosc.eu/?path=/docs/experimental-surfacevis--docs) (experimental)
 
+### Constants
```

Comment:
Still planning on improving/splitting up this file but thought I'd at least document the constants here for now.

Reply:
Do you think it is worth documenting the constants ? We don't expect consumers from the `lib` to use them, no ?

Reply:
Fair enough, I'll remove them.

Reply:
This being said, giving the consumers the limits of the camera FOV would perhaps be helpful ?

Reply:
I've added two constants: `Z_MIN` and `Z_MAX`, and documented them. It's clearer indeed, including in the code/comments.